models/oneinch/ethereum/uniswap/oneinch_uniswap_v3_ethereum_trades.sql

```diff
@@ -0,0 +1,377 @@
+{{  config(
+        schema='oneinch_uniswap_v3_ethereum',
+        alias='trades',
+        partition_by = ['block_date'],
+        on_schema_change='sync_all_columns',
+        file_format ='delta',
+        materialized='incremental',
+        incremental_strategy='merge',
+        unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'trace_address']
+    )
+}}
+
+{% set project_start_date = '2021-11-07' %} --for testing, use small subset of data
+{% set burn_address = '0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee' %} --according to etherscan label
+{% set blockchain = 'ethereum' %}
+{% set blockchain_symbol = 'ETH' %}
+
+WITH uniswap AS
```

Comment:
this model is also producing one outlier tx, but unlike the other model with forced exclusion on tx's, even etherscan is showing the high `amount_usd`
https://etherscan.io/tx/0x886425a6236f1c7aaa91e622f8ec2987c7e1a85164206fa3c8cc381a32df9248

for now, leaving the tx included